### DISCUSSION TITLE
Local cache for initiator

### DISCUSSION THREAD

**Discussion by boxerab** [open-iscsi/open-iscsi#489] [General]
Hello, I would like to find a way of reducing network traffic between my initiator and target, but caching data locally on the initiator. I found [this issue](https://github.com/open-iscsi/open-iscsi/issues/199) on the topic. I looked into `bcache` or `dm-cache` but these belong on target. Is there a local cache solution for the initiator?  Any insight here would be greatly appreciated.

**Comment by gonzoleeman** [comment]
Well, if you are attaching to a disk block device (as usual), then the block subsystem will cache some of changes I believe. Other than that, I don't know of anything.

Sorry to not notice this question for so long!

**Comment by boxerab** [comment]
Thanks for your reply. Going to closes this as I am no longer working on this problem.
